### ISSUE TITLE
Too much quotation in generated configurations in .env.public

### ISSUE THREAD

**Issue by cmungall** [linkml/linkml-project-cookiecutter#116]
https://github.com/linkml/linkml-project-cookiecutter/blob/4e062ec2471c7110af2f2ae4fc4a016fcb188bd3/%7B%7Bcookiecutter.project_name%7D%7D/.env.public#L5-L10

The quotes don't function like normal quotes, they end up in the string

**Comment by sbliven** [comment]
I ran into this too. Makefiles expect variables not to be quoted. The default `.env.public` results in errors like
```
bash: -c: line 1: unexpected EOF while looking for matching `"'
make: *** [compile-sheets] Error 2
```

**Comment by noelmcloughlin** [comment]
I will have a look at this.

**Comment by amc-corey-cox** [comment]
Here is my full error message:

Writing lock file

Installing the current project: sepio_linkml (0.0.0.post2.dev0+842b967)
mkdir -p src/"sepio_linkml"/datamodel
poetry run sheets2linkml --gsheet-id "19Rp5DLFS3a1Yvvw84mPupgEkCMFRKax4LXCcwX84Vbg" "release_subset_v1" > "src/sepio_linkml/schema//release_subset_v1.yaml.tmp && mv "src/sepio_linkml/schema//release_subset_v1.yaml.tmp "src/sepio_linkml/schema//release_subset_v1.yaml
bash: -c: line 1: unexpected EOF while looking for matching `"'
make: *** [Makefile:116: compile-sheets] Error 2

**Comment by amc-corey-cox** [comment]
This looks like the quotes are somehow being dropped on the ends of variables when we run sheets2linkml. I got this when I ran `make setup`.

**Comment by amc-corey-cox** [comment]
This is the offending line in the Makefile:

```
$(RUN) sheets2linkml --gsheet-id $(SHEET_ID) $(SHEET_TABS) > $(SHEET_MODULE_PATH).tmp && mv $(SHEET_MODULE_PATH).tmp $(SHEET_MODULE_PATH)
```
I don't know why it is eating the `"` at the end of these variables from the variables defined at the top of the Makefile but it appears to be when we append text to the variable, so the statements `$(SHEET_MODULE_PATH).tmp` and `$(SHEET_MODULE_PATH).tmp` don't have closing `"`. Also, `$(SHEET_MODULE_PATH)` which is defined in the header as `$(SOURCE_SCHEMA_DIR)/$(SHEET_MODULE).yaml` has the same problem with missing the closing `"`.

**Comment by amc-corey-cox** [comment]
Removing quotes from the variable definition for `LINKML_SCHEMA_SOURCE_PATH` in env.public solves the issue and allows `make setup` to run the line without issues related to `"`. I'm not very familiar with Makefile so I'm not sure how to solve this in the current infrastructure but I'd be happy to help.

**Comment by noelmcloughlin** [comment]
Thanks for feedback - unfortunately I am intermittent contributor (and my work laptop cannot access public GitHub, so I need to grab my neglected personal laptop). This issue was first raised in #106 so needs solution.

Would you be wiling to raise a PR to remove all quotes from  `.env.public` file?  @amc-corey-cox 

This is acceptable regression as that file is my idea and needs incremental fixing.
I raised wider discussion here: https://github.com/orgs/linkml/discussions/2296

**Comment by amc-corey-cox** [comment]
@noelmcloughlin Sure, I'm happy to help out on this. I'll try just implementing your suggestion and see what we break.

**Comment by sbliven** [comment]
Makefile quoting is a mess. I think the normal suggestion is to not use quotes if there aren't any spaces in the variable (definitely true for the google vars, and likely true for the path)

**Comment by amc-corey-cox** [comment]
Looking a little deeper at this, this is actually a make file, not an environment file or a Bash script file. I think we should be more clear in how we define this.

At minimum, I will include a statement in the file that this is a make-file. I would prefer to change the name from .env.public to .vars.public.mk but I'm concerned this might cause unnecessary breakage for users using cruft to update their cookie-cutters.

For now, I'll remove the excess quotation and make a notation within the file that states it is a make-file not an env file or a bash script (it kinda looks like it could be either).

**Comment by noelmcloughlin** [comment]
Hi !!
The file originated from me and is not community driven request - i.e. nobody requested a make file with variables.
I was testing this project with different models and encountered various problems so I raised few Issues and lot of PRs.
I focused on setting environment variables manually and/or using that file and everything eventually seemed to work.
However, issues have emerged since -- so I think I need to rethink entire solution.

Feel free to rename file - I think nobody else cares at this point - they just want bug fixed I guess so that is why I suggested removing quotes as first step.
